### PR TITLE
Add PyTorch integration tests with FDS

### PR DESCRIPTION
**Pull request by adam-narozniak**

### CODE REVIEW

File: datasets/integration_tests/pytorch/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+version = "0.1.0"
+description = "Flower Datasets with PyTorch"
+authors = ["The Flower Authors <hello@flower.dev>"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+flwr = "^1.5"
+torch = "^1.12.0"
+torchvision = "^0.14.1"
+flwr_datasets = { path = "./../../", develop = true, extras = ["vision"] }
```
Comment (danieljanes) [inline datasets/integration_tests/pytorch/pyproject.toml:16]:
```suggestion
flwr-datasets = { path = "./../../", develop = true, extras = ["vision"] }
```

File: datasets/integration_tests/pytorch/pytorch_test.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+#    and then normalizes it.
+@parameterized_class(
+    [
+        {"dataset_name": "cifar10", "test_split": "test", "transforms": ToTensor()},
+        {"dataset_name": "cifar10", "test_split": "test", "transforms": Compose(
+            [ToTensor(), Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))]
+        )},
+    ]
+)
+class FDSToPyTorchCorrectUsage(unittest.TestCase):
```
Comment (danieljanes) [inline datasets/integration_tests/pytorch/pytorch_test.py:48]: We usually CamelCase abbreviations:
```suggestion
class FdsToPyTorchCorrectUsage(unittest.TestCase):
```

File: datasets/integration_tests/pytorch/pyproject.toml (hunk 1 above)
Comment (danieljanes) [reply datasets/integration_tests/pytorch/pyproject.toml:16]: Also, should we move this to right after `flwr`?

**Review by danieljanes — CHANGES_REQUESTED** [review]
Let's make the same changes in the other two integration test PRs (if applicable)

File: datasets/integration_tests/pytorch/pyproject.toml (hunk 1 above)
Comment (adam-narozniak) [reply datasets/integration_tests/pytorch/pyproject.toml:16]: Yes, I'll do that

Comment (adam-narozniak) [reply datasets/integration_tests/pytorch/pyproject.toml:16]: Actually, flwr shouldn't be listed there. I removed it.

**Review by charlesbvll — APPROVED** [review]
LGTM!

**Review by tanertopal — APPROVED** [review]

**Pull request merged**